dali/kernels/test/mat2tensor.h

```diff
@@ -64,6 +66,30 @@ TensorView<StorageCPU, T, ndim> view_as_tensor(cv::Mat &mat) {
   return { mat.ptr<T>(0), tensor_shape<ndim>(mat) };
 }
 
+
+template<typename T = uint8_t, int ndims = 3>
+std::pair<TensorView<StorageGPU, T, ndims>, memory::KernelUniquePtr<T>>
+view_as_tensor_gpu(const cv::Mat &mat) {
+  auto tvcpu = kernels::view_as_tensor<const T, ndims>(mat);
+  auto mem = kernels::memory::alloc_unique<T>(kernels::AllocType::Unified,
+                                              mat.cols * mat.rows * mat.channels());
```

Comment:
It is assumed that mat is uint8.
mat.total() * mat.elemSize() ?

Reply:
It's OK; `alloc_unique<T>` takes number of T elements, not size in bytes.

Reply:
So only mat.total()? Because mat.rows and mat.cols can be `-1` for more dimensions.